rules/windows/defense_evasion_execution_lolbas_wuauclt.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2020/10/13"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/13"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies Execution via abusing the Windows Update Client to load an arbitrary Image."
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "ImageLoad via Windows Update Client"
+references = ["https://dtm.uk/wuauclt/"]
+risk_score = 47
+rule_id = "edf8ee23-5ea7-4123-ba19-56b41e424ae3"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+timeline_id = "76e52245-7519-4251-91ab-262fb1a1728c"
+type = "eql"
+
+query = '''
+
+process where event.type in ("start", "process_started") and
+  (process.pe.original_file_name == "wuauclt.exe" or process.name == "wuauclt.exe") and
```

Comment:
pending `process.pe.original_file_name` being mapped into `winlogbeat`

Reply:
removed blocked for process.pe.ofn